src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends AbstractFigure {
+    private final double side;
+
+    public Square(double side, String color) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create a separate variable for the 'side' and pass it to the constructor for better code readability.